### PR TITLE
Some more code improvements

### PR DESCRIPTION
**Pull request by sosiska**
Some more code improvements.

### CODE REVIEW

**Comment by Morganamilo** [comment]
If you're gonna keep going through the code I would suggest you keep at it and make one bigger than a bunch of tinny ones.

**Comment by sosiska** [comment]
OK, i will look for. Then i will squash commits

**Comment by sosiska** [comment]
I think that all(no more suitable if-else-if-else chains 🙃). Look at the separate commits to see what I have changed. Then, if there are OK, I will squash them to big one.

File: config.go [hunk 1]
```diff
@@ -288,8 +288,8 @@ func continueTask(s string, cont bool) bool {
 
 	fmt.Print(bold(green(arrow)+" "+s), bold(postFix))
 
-	len, err := fmt.Scanln(&response)
-	if err != nil || len == 0 {
+	i, err := fmt.Scanln(&response)
```
Comment (Jguer) [inline config.go]: Maybe `len` was not the best descriptor for the return, but I think `i` is not much better. Maybe the `n` which is the same as the function header `func Scanln(a ...interface{}) (n int, err error)` would be a better choice.

Comment (Morganamilo) [reply config.go]: >It returns the number of items successfully scanned. If that is less than the number of arguments, err will report why.

Maybe we don't even need the variable.

Comment (Jguer) [reply config.go]: True, maybe it can even be removed (if it's like that it would never evaluate it anyway)

Comment (sosiska) [reply config.go]: You are right, judging by the function description, the second comparison will not happen. Variable removed in 0347d1d3d69671a7f482193a1b5e25ef5bd0afc8

**Comment by sosiska** [comment]
Squashed.

File: depCheck.go [hunk 2]
```diff
@@ -9,7 +9,7 @@ import (
 	alpm "github.com/jguer/go-alpm"
 )
 
-func (dp *depPool) checkInnerConflict(name string, conflict string, conflicts mapStringSet) {
```
Comment (Morganamilo) [inline depCheck.go]: I'm not sure what's more idiomatic for go but my personal opinion is to only skip the return types if all returns are the same type.

Comment (sosiska) [reply depCheck.go]: OK. Combined parameters in some functions are rejected in 14ac7562981a20bbc0a91c7fcdafc7a1c6b6ab9c.

**Comment by sosiska** [comment]
Ping

**Comment by Jguer** [comment]
I'm ok with it and merging, thanks for the contribution @sosiska

**Pull request merged**